penguin/web/server.py

```
@@ -244,4 +275,4 @@ def start_server(
 
 
 if __name__ == "__main__":
-    exit(main())
+    exit(main(sys.argv[1:]))
```

Ruff (0.15.11)

[warning] Use `sys.exit()` instead of `exit`

Replace `exit` with `sys.exit()`

(PLR1722)